start

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-node chesster.js $1
+node chesster.js $1 >> stdout_log 2>> stderr_log
```

Comment:
Hm, I'm ok with that - any logging system your are familiar with? This goes above and beyond the requirements, but could be good to tag the beginning of each line with a timestamp and file?

`2016-05-14T16:13:58+00:00 - stdout: something something something `